src/main/restartDialog.js

```diff
@@ -0,0 +1,129 @@
+/*!
+GPII Application
+Copyright 2016 Steven Githens
+Copyright 2016-2017 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+"use strict";
+
+var fluid   = require("infusion");
+var gpii    = fluid.registerNamespace("gpii");
+var ipcMain = require("electron").ipcMain;
+
+require("./dialog.js");
+
+/**
+ * A component that serves as simple interface for communication with the
+ * electron `BrowserWindow` restart dialog.
+ */
+fluid.defaults("gpii.app.dialog.restartDialog.channel", {
+    gradeNames: ["fluid.component"],
+
+    events: {
+        // onClosed: null
```

Comment:
If this grade is intended for use as a mixin grade for another, specify what that base grade is (holding definition of onClosed). In these cases we also tend to lead out the "gradeNames" specification (if the grade cannot function as a self-sufficient component)